dev/bots/test.dart

```diff
@@ -882,6 +882,7 @@ Future<void> _runIntegrationTests() async {
         await _runDevicelabTest('flutter_create_offline_test_windows');
       } else if (Platform.isMacOS) {
         await _runDevicelabTest('flutter_create_offline_test_mac');
+        await _runDevicelabTest('plugin_lint_mac');
 // TODO(jmagman): Re-enable once flakiness is resolved.
```

Comment:
Is there an issue filed for this TODO?

Reply:
It's https://github.com/flutter/flutter/issues/37525.  https://github.com/flutter/flutter/issues/37519 is a really good idea.